mlir/test/Dialect/SCF/loop-pipelining.mlir

```diff
@@ -764,11 +764,44 @@ func.func @stage_0_value_escape(%A: memref<?xf32>, %result: memref<?xf32>, %ub:
 //      NOEPILOGUE:     memref.load %[[A]][%[[IV3]]] : memref<?xf32>
 //      NOEPILOGUE:   scf.yield %[[V2]], %[[L3]] : f32, f32
 
-// In case dynamic loop pipelining is off check that the transformation didn't
```

Comment:
Do we want to keep a test checking this case (not pipelining when dynamic loop support is turned off?)

Reply:
I would need to add a new switch to TestSCFUtils and probably a new test file so we don't run all the tests again without dynamic loop support. Or perhaps add it to the annotate run? Is that acceptable?

Reply:
Hmm, not sure if this is worth the effort to validate that the transformation is *disabled*. I think I'm OK if you'd want to skip it.